Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient.h

```diff
@@ -114,104 +114,12 @@ typedef NS_ENUM(NSUInteger, BUYStatus) {
 };
 
 /**
```

Comment:
This header should not need BUYCollectionSort, BUYClientCustomerAccessToken, BUYDataStatusBlock, BUYStatus, and several of the @class forward declarations